test_case.py

```diff
@@ -0,0 +1,33 @@
+import django
+from django.conf import settings
+from django.core.management import call_command
+from django.test import TestCase
+
+from test_settings import TEST_SETTINGS
+
+
+def setup_django():
+    if not settings.configured:
+        settings.configure(**TEST_SETTINGS)
+        # https://docs.djangoproject.com/en/1.11/topics/settings/#calling-django-setup-is-required-for-standalone-django-usage
+        django.setup()
+        # applying migrations manually is needed only for running from within GUI
+        call_command('migrate')
+    else:
```

Comment:
is else branch required?

Reply:
There is explanation below

Reply:
The question was why we need the else pass to create a comment. But I understand now that. `else-pass` is a comment itself rather than a codeblock

Reply:
Ah, got it, you right then.

Reply:
@akoidan I do that pretty often in cases when I find my if-elses not self-explanatory enough. If that's against team code standards please let me know, I can adjust :)